description: avoid building chromium's installer, which is unused in debian
author: Michael Gilbert <mgilbert@debian.org>

--- a/BUILD.gn
+++ b/BUILD.gn
@@ -147,8 +147,6 @@ group("gn_all") {
 
   if (!is_castos) {
     deps += [
-      "//chrome/installer",
-      "//chrome/updater",
     ]
   }
 
